src/app/api/services/dot-render-html/dot-render-html.service.ts

```diff
@@ -85,3 +103,9 @@ export class DotRenderHTMLService {
         return pageModeString[pageMode];
     }
 }
+
+class PageOptions {
```

Comment:
DotPageOptions?

Reply:
Why class and no interface?

Reply:
very much time coding just in java hahahahahahha